custom/panel_templates/Default/integrations/discord/discord.tpl

```diff
@@ -84,6 +84,15 @@
                                         <input class="form-control" type="number" name="discord_guild_id"
                                             id="inputDiscordId" value="{$DISCORD_GUILD_ID_VALUE}">
                                     </div>
+                                    <div class="form-group">
```

Comment:
We should either have a tool tip here to tell them where to setup discord oauth, or not display the `"oauth"` option unless it is enabled

Reply:
Well that setting is temporially until the integrations settings page gets a proper settings page with allows integrations to add fields and handle it

adding tool tip here being same as adding tool tip for Discord Bot aswell to see if its setup or not, default method is even bot with may not be setup

And we still have the option as you discussed earlier to remove the Bot linking, there is alot of people who copy/paste the command into discord, but the commands system not detect it as they write it as text instead so they're leaking the code so other people can fast link their account before they do as many don't understand they need to write /verify manually